apps/tup-ui/src/main.global.css

```diff
@@ -1,3 +1,17 @@
+/* To use new c-nav component not yet in TACC/Core-CMS core-styles.base.css */
+/* TODO: Remove when TACC/Core-CMS loads TACC/Core-Styles v2.13+ */
+/* CAVEAT: Duplicates older, unchanged, global c-nav styles */
+@import url("@tacc/core-styles/dist/components/c-nav.css");
+
+/* To use s-affixed-input-wrapper which had been unused outside TACC/Core-CMS */
+/* TODO: Remove when TACC/Core-CMS loads TACC/Core-Styles v2.14+ */
+@import url("@tacc/core-styles/dist/trumps/s-affixed-input-wrapper.css");
+
+/* To use new form styles in core-styles.base.css */
+/* TODO: Remove when TACC/Core-CMS loads TACC/Core-Styles v2.15+ */
+@import url("@tacc/core-styles/dist/elements/form.css");
+@import url("@tacc/core-styles/dist/trumps/s-form.css");
```

Comment:
Available since #279 (instead of using minimal `s-form` in tup-ui from previous commits).